internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -19,6 +19,7 @@ const (
 	lbWebSvcRulePriorityGeneratorPath = "custom-resources/alb-rule-priority-generator.js"
 	desiredCountGeneratorPath         = "custom-resources/desired-count-delegation.js"
 	envControllerPath                 = "custom-resources/env-controller.js"
+	lbWebSvcType                      = "Load Balanced Web Service"
```

Comment:
i'm wondering if it makes sense to move the workloadType constants into one `const` block defined in stack/workload.go? So instead of having `lbWebSvcType` in lb_web_svc.go and `backendSvcType` in backend_svc.go, we have
```
// Types of workload created by the stack.
const (
 lbWebSvcType    = "Load Balanced Web Service"
 backendSvcType  = "Backend Service"
)
```
in stack/workload.go